### PR TITLE
Install zstd on CircleCI linux

### PR DESCRIPTION
**Pull request by riversand963**

### CODE REVIEW

File: .circleci/config.yml [hunk 1]
```diff
@@ -147,6 +147,13 @@ commands:
           command: |
             sudo apt-get update -y && sudo apt-get install -y libsnappy-dev zlib1g-dev libbz2-dev liblz4-dev libzstd-dev
 
+  install-streaming-compress-libs:
```
Comment (gitbw95) [inline .circleci/config.yml:150]: n00b: 
1. Does WAL compression only work over streaming compress libs?
2. Do we have lz4 streaming compress lib on the circleci box? If so, can WAL compression us the lz4 one?

Comment (riversand963) [reply .circleci/config.yml:150]: According to https://github.com/facebook/rocksdb/blob/main/util/compression.h#L526, ZSTD so far is the only lib that may support streaming compression.

Comment (riversand963) [reply .circleci/config.yml:150]: > Does WAL compression only work over streaming compress libs?

Yes

**Review by jay-zhuang — APPROVED** [review]
LGTM

**Comment by riversand963** [comment]
Thanks @jay-zhuang for the review

**Pull request closed without merging**